components/org.wso2.carbon.identity.api.idle.account.identification/org.wso2.carbon.identity.api.idle.account.identification.v1/src/main/resources/META-INF/cxf/idle-account-identification-cxf.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is added again with a different path. Please validate what is the actually required file and remove the other one.

Reply:
Addressed with commit e52d2135a2cab52434b3a4ae45415981e1ab9011